open-api/rest-catalog-open-api.yaml

```diff
@@ -3014,28 +3060,26 @@ components:
           additionalProperties:
             type: string
 
-    PlanTableResult:
+    ScanTasks:
       type: object
-      description: If the plan has not finished return a `plan-id`. 
-        If finished, the response will contain a list of `FileScanTask`, a list of `PlanTask`, or both.
+      description:
+        Scan and planning tasks for server-side scan planning
+
+        - `plan-tasks` contains opaque units of planning work
+        - `file-scan-tasks` contains a partial list of table scan tasks
+        - `delete-files` contains delete files referenced by file scan tasks
+
+        Each plan task must be passed to the fetchScanTasks endpoint to fetch
+        the file scan tasks for the planning task.
```

Comment:
```suggestion
        Each plan task must be passed to the fetchScanTasks endpoint to fetch
        the relevant file scan tasks associated with it.
```

Reply:
I think this is saying the same thing, but more succinctly. What is the rationale for your edit?

Reply:
I think we were just using "plan task" and then planning task in the sentence so offered the reword since it seems redundant . Should have marked this as [nit] as im ok with going with the original as well.

Reply:
I updated this to "plan task"